internal/display/infowindow.go

```diff
@@ -146,25 +138,19 @@ func (i *InfoWindow) displayBuffer() {
 
 		width := 0
 
-		char := ' '
 		switch r {
 		case '\t':
-			ts := tabsize - (totalwidth % tabsize)
-			width = ts
+			width = tabsize - (totalwidth % tabsize)
+			for j := 1; j < width; j++ {
```

Comment:
We should start from 0, not from 1? (assuming that the overall approach of this PR is correct at all)